predictoor/pdr-faq.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Predictoor FAQ
+description: Frequently Asked Questions about Predictoor
+---
+
+## Predictoor FAQ
+<details>
+
+<summary>How Predictoor compare to prediction markets? Like Predictoor, they use \$ for accountability.</summary>
```

Comment:
`How Predictoor` -> `How does Predictoor`

Why: fix grammar. (BTW the blog had this error too. I just fixed it there too.)

Reply:
Cheers, done!